tests/integration/targets/filter_split_pem/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
```

Comment:
total  nit and I don't care if it stays, but is this warning still useful? I don't really see it in anything new

Reply:
I don't know how useful this really is, but I think it's better to keep it :)